prody/sequence/motif.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""This module gets PDB code from MOTIF."""
+
+import argparse
+import re
+
+import requests
+from prody import LOGGER
+
+__author__ = "Mariusz Konstanty"
+
+__all__ = ["getPdbFromMotif", "expasySearchMotif", "_expasyTextToStructure"]
+
+DATABASES = {
+    "sp": "Swiss-Prot",
+    # "rs": "RefSeq",
+    "pdb": "PDB",
+    "tr": "TrEMBL",
+    "local": "local database",
+}
+MOTIF_PATTERN = (
+    r"^([ARDNCEQGHILKMFPSTWYVx]*"
+    r"(\[[ARDNCEQGHILKMFPSTWYV]+\])*"
+    r"({{[{{ARDNCEQGHILKMFPSTWYV}}]+}})*"
+    r"([ARDNCEQGHILKMFPSTWYVx]\(\d(,\d)?\))*)+$"
+)
+MOTIF_MATCHER = re.compile(MOTIF_PATTERN)
+
+
+def getPdbFromMotif(motif: str, database: str) -> str:
+    if database not in DATABASES:
+        raise ValueError(f"Database must be one of: {DATABASES.keys()}.")
```

Comment:
Please don't use f strings. They don't work with older python versions.

Reply:
> Everything look much better now. Thanks.
> 
> Still if you could replace f strings with older types of formatting, that would be great. We need compatibility with python 2.7 for building the prody website.

hello James,
python 2.7 and 3.5 are end of life, could you please consider abaddon backward compatibility for those versions? - it's not just about f-strings.

Reply:
No, unfortunately we can't. We can drop 3.5 for sure but not 2.7

If you (or anyone else) know how to build websites with sphinx and can help reconfigure ours then we might be able to but otherwise no.